ghostscript.yaml

```diff
@@ -0,0 +1,128 @@
+package:
+  name: ghostscript
+  version: 10.01.2
+  epoch: 0
+  description: Interpreter for the PostScript language and for PDF
+  copyright:
+    - license: AGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - autoconf
+      - automake
+      - cups-dev
+      - expat-dev
+      - freetype-dev
+      - gtk-2.0-dev
+      - jbig2dec-dev
+      - lcms2-dev
+      - libjpeg-turbo
+      - libpng-dev
+      - libtool
+      # - openjpeg-dev
```

Comment:
Is openjpeg needed?  I noticed that we're adding the melange config below but `openjpeg-dev` is commented out above.

Reply:
yeah, I'll send a complementary fix PR for adding openjpeg-dev afterwards we merge this PR since I'm adding packages in single PR but I can separate them but this time I need to wait openjpeg to be built first, then I can add ghostscript package, WDYT?

Reply:
As long as openjpeg comes before ghostscript in the `packages.txt` which it does, it should be fine to uncomment the dependency above and we can make sure CI passed?

Reply:
great, lets give it a try!

Reply:
it worked ✅